neet/boolean/randomnet.py

```diff
@@ -66,7 +66,15 @@ def _random_binary_states(n, p):
     return set(tuple(state) for state in StateSpace(n) if random.random() < p)
 
 
-def _random_logic_fixed_connections(logic_net, ps):
+def _external_nodes(logic_net):
+    externals = set()
+    for idx, row in enumerate(logic_net.table):
+        if len(row[0]) == 1 and row[0][0] == idx:
```

Comment:
Is this equivalent to `if row[0] == [idx]:`?

Reply:
That's right. I'll change that.

Reply:
And I need to check whether the condition is `{'1'}`.